### DISCUSSION TITLE
Can the knife in the logo be removed/changed?

### DISCUSSION THREAD

**Discussion by firatoezcan** [crablang/crab#91] [Ideas]
I'm the last one to not find it funny but I think maybe something like a water gun or an eyepatch would be less "knife in logo" 🦀

**Comment by onehex** [comment]
knife is a killer move by crablang.
i support knife.

**Reply by i007c** [reply]
im not a crab or rust user but i like knives

**Comment by Traneptora** [comment]
![image](https://github.com/crablang/crab/assets/1342577/bcbbcda1-dbe9-488f-9b25-011cf8ca21ab)

**Comment by noctisatrae** [comment]
Love this picture!
